sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -395,3 +395,12 @@ def test_rfe_allow_nan_inf_in_x(cv):
         rfe = RFE(estimator=clf)
     rfe.fit(X, y)
     rfe.transform(X)
+
+
+def test_multioutput():
```

Comment:
I imagine the same should work with RFECV? If so please parametrize this test with,

```py
@pytest.mark.parametrize('ClsRFE', [RFE, RFECV])
def test_multioutput(ClsRFE):
    ...
    rfe_test = ClsRFE(clf)
```

Reply:
Yeah this will work with REFCV I will also add this in the test. Thank you so much @rth : )